src/core/lib/event_engine/posix_engine/event_poller_posix_default.cc

```diff
@@ -58,5 +62,11 @@ PosixEventPoller* GetDefaultPoller(Scheduler* scheduler) {
   return poller;
 }
 
+#else  // GRPC_POSIX_SOCKET_TCP
+
+PosixEventPoller* GetDefaultPoller(Scheduler* /*scheduler*/) { return nullptr; }
```

Comment:
Change name to MakeDefaultPoller, MakeEpoll1Poller, MakePollPoller, etc

Reply:
Done.